internal/services/mysql/mysql_flexible_server_resource.go

```diff
@@ -243,6 +243,12 @@ func resourceMysqlFlexibleServer() *pluginsdk.Resource {
 				ValidateFunc: privatezones.ValidatePrivateDnsZoneID,
 			},
 
+			"public_network_access_enabled": {
```

Comment:
One more note here, Microsoft is adding more options to `public_network_access_enabled` so a boolean won't work when that change gets added. We should rename this to `public_network_access` and have it be a string

Reply:
Roger, thanks! Will do.